extensions/drupal-org/src/utils/search-api/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface ApiItem {
```

Comment:
Do you see the value in having a single `types` file containing all the types? I think some of the types such as `DrupalVersions` might be useful for future commands. Then again, we can refactor at that time.

Reply:
Yeah, I sure see the value in that. Maybe it would be better to have one general type file, while keeping the types specific to one command separate. What do you think about that?

Reply:
I'm currently working on another feature for the app, so I could do the refactor in the next PR if that's ok with you.

I implementing search for projects on drupal.org as mentioned in README.md if you're curious as to which feature I'm building :-).

Reply:
> while keeping the types specific to one command separate. What do you think about that?

That sounds good. And yes, we can refactor this later. And I'm looking forward to the project search feature as well. 🎉